docs/index.md

```diff
@@ -1,25 +1,49 @@
-# Introduction
+# Introduction to `cfgov-refresh`
 
 This is the documentation for the `cfgov-refresh` project, a redesign of the [www.consumerfinance.gov](https://www.consumerfinance.gov) website. It is organized thematically in order to create a central repository for all information pertaining to cfgov-refresh.
 
-# Disclaimer
+## Disclaimer
 
 **This project is a work in progress.** Nothing presented in this repo—whether in the source code, issue tracker, or wiki—is a final product unless it is marked as such or appears on [www.consumerfinance.gov](https://www.consumerfinance.gov). In-progress updates may appear on [beta.consumerfinance.gov](https://beta.consumerfinance.gov).
 
-# Technology stack
+## Technology stack
 
 The standard technology stack for development of cfgov-refresh within the CFPB consists of the following base:
 
 - macOS
 - [Homebrew](https://brew.sh) - package manager for installing system software on OSX
-- Python and PIP (Python package manager)
-- [Jinja templates](http://jinja.pocoo.org) for front-end rendering
-- [Wagtail CMS](https://wagtail.io) for content administration
-- Dependencies, listed below
+- [Python 2.7](https://docs.python.org/2.7/) and [pip (Python package manager)](https://pip.pypa.io/en/stable/user_guide/)
+- [Jinja2 templates](http://jinja.pocoo.org/docs/2.10/) for front-end rendering. See [`requirements/libraries.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/libraries.txt) for version.
+- [Wagtail CMS](https://wagtail.io) for content administration. See [`requirements/wagtail.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/wagtail.txt) for version.
+- [PostgreSQL](https://www.postgresql.org/) and [Psycopg](http://initd.org/psycopg/) for our database. See [`requirements/postgres.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/postgres.txt) for version.
```

Comment:
Technically `postgres.txt` doesn't define the version of Postgres that we run (currently 10.x in production), but only the version of Psycopg. It would be good to document the recommended version of Postgres, maybe here like we do for Python 2.7.

Reply:
Could you add a suggestion here for how to word this? I found the postgres vs psychopg thing confusing (their own documentation didn't help!) so it would be extremely valuable for you to add some clarity here!!

Reply:
Sure! PostgreSQL is the database we run in production and that we recommend running locally. Although we don't currently use any Postgres-specific database features AFAIK, we've discussed doing so often and likely will in future. In production we run Postgres 10.5 at the moment, although the servers get periodic upgrades.

Psycopg is the Python library that lets Python talk to Postgres. Originally it was in its own requirements file to avoid installing it when running on MySQL (or SQLite, as we do on Jenkins). We could consider merging it into libraries.txt.

For now, I might recommend:

> We recommend using [PostgreSQL 10](https://www.postgresql.org/docs/10/index.html) as the database backend for this project. We use [Psycopg]((http://initd.org/psycopg/)) as the Python adapter for Postgres. See [`requirements/postgres.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/postgres.txt) for current version.

Reply:
I adapted your descriptions and added to the doc!